python/ray/util/client/server/server.py

```diff
@@ -768,7 +766,7 @@ def main():
         choices=["proxy", "legacy", "specific-server"],
         default="proxy")
     parser.add_argument(
-        "--redis-address",
+        "--address",
```

Comment:
I'm not very comfortable with this API change. Can you double-check that it's not a public API?

Reply:
Users do not need to launch Ray client server directly anymore. Also if we were to keep `--redis-address`, once GCS bootstrapping is enabled, `--redis-address` will silently become a no-op too.